nvflare/private/fed/server/fed_server.py

```diff
@@ -174,7 +174,9 @@ def deploy(self, args, grpc_args=None, secure_train=False):
                 root_certificates=root_ca,
                 require_client_auth=True,
             )
-            self.grpc_server.add_secure_port(target, server_credentials)
+            port = target.split(":")[1]
+            tmp_target = f"0.0.0.0:{port}"
+            self.grpc_server.add_secure_port(tmp_target, server_credentials)
```

Comment:
why are we ignoring the "host" specified in target?

Reply:
The same as the above.

Reply:
got it, Should we also change other places using this "target"?

and maybe we want to change the "fed_server.json" to get rid of the host